ui/js/controllers/perf/graphs.js

```diff
@@ -1105,7 +1105,7 @@ perf.controller('TestChooserCtrl', ['$scope', '$uibModalInstance',
                                             projectName: test.projectName,
                                             signature: test.signature });
                                     });
-                                $scope.loadingTestData = false;
+                                $scope.$apply($scope.loadingTestData = false);
```

Comment:
I guess this works, but it's a little weird. Any reason you can't just run the $scope.apply() by itself?

Reply:
Sure, that's fine, too.  I've used ``$timeout`` for things like this in the past, and suppose I got in the habit of doing that.  So no particularly good reason to do it that way.  :)  Fixed.